packages/ilios-common/app/styles/ilios-common/mixins/ilios-list.scss

```diff
@@ -19,16 +19,21 @@
   background-color: var(--white);
   border: 1px solid var(--cultured-grey);
   border-radius: 3px;
-  padding: 1em 2em;
+  padding: 1em;
   width: 80%;
 
   li {
     font-weight: bold;
+
+    &:last-of-type {
+      margin-bottom: 0;
+    }
   }
 
   ul {
     margin-left: 1em;
 
+    /* stylelint-disable-next-line no-descending-specificity */
```

Comment:
This points to an issue in the overall structure of the styles here and usually causes problems. Refactor to remove the need to this please!